TaxonomyService.md

```diff
@@ -0,0 +1,61 @@
+Taxonomy Service
+=============
+
+Table of Contents
+_________
+- [Request Information](#request-information)
+- [Sample Response](#sample-response)
+- [Available Taxonomies](#available-taxonomies)
+- [Versioning](#versioning)
+
+
+
+#Request Information
+
+
+HTTP method: GET
+Parameters (query/form):
+-        No Parameters
```

Comment:
"No Parameters" isn't true -- we just don't take any *query* parameters. The example should also be a valid request - what you call an example is more like a demonstration of the format. I would rewrite this like:

HTTP method: GET
Parameters: The only parameter accepted by this service is the name of the desired taxonomy, provided as part of the URL path.
```
https://api.careerbuilder.com/core/taxonomy/{taxonomyName}
```
Example: https://api.careerbuilder.com/core/taxonomy/onet17